__tests__/app.test.js

```diff
@@ -628,4 +628,72 @@ describe('demo routes', () => {
     });
   });
 
+  it('/DELETE with wrong user, verify error message', async() => {
```

Comment:
nice work, all.  Love that you went the extra step to verify that the wrong owner could not delete, instead of just testing that it can be deleted.